appinfo/info.xml

```diff
@@ -3,7 +3,14 @@
 	  xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>contacts</id>
 	<name>Contacts</name>
-	<description>The new and improved app for your Contacts.</description>
+	<summary>A contacts app for Nextcloud.</summary>
```

Comment:
I know it is a minor thing but all other apps **don't** use a full stop at the end of the sentence - I think it would look weird if contacts (and calendar) use a dot while other apps don't. Do you plan to change it accordingly for others apps too? Otherwise I would suggest we remove it.